test/225_dns_subdomain_test.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/bash
+
+. ./config.sh
+
+C1=10.2.54.34
+C2=10.2.54.91
+C3=10.2.54.92
+DOMAIN=foo.bar
+SUBDOMAIN1=sub1.$DOMAIN
+SUBDOMAIN2=sub2.$DOMAIN
+
+start_suite "Resolve names in custom domain with subdomains"
+
+weave_on $HOST1 launch --log-level=debug --dns-domain $DOMAIN.
+
+weave_on $HOST1 run --with-dns $C1/24 --name=c1 -h foo.$SUBDOMAIN1 -t $NONBROKENRESOLVER_IMAGE /bin/sh
+weave_on $HOST1 run --with-dns $C2/24 --name=c2 -h foo.$SUBDOMAIN2 -t $NONBROKENRESOLVER_IMAGE /bin/sh
+weave_on $HOST1 run --with-dns $C3/24 --name=c3 -h tre.$SUBDOMAIN1 --dns-search=$SUBDOMAIN2 -t $NONBROKENRESOLVER_IMAGE /bin/sh
```

Comment:
why not `start_container_with_dns_on` for all the above? That launches containers based on `aanand/docker-dnsutil`, which in turn is based on ubuntu.